src/raftstore/store/snap.rs

```diff
@@ -1316,6 +1332,10 @@ impl SnapManager {
             receiving_count: receiving_cnt,
         }
     }
+
+    pub fn get_limiter(&mut self) -> &mut RateLimiter {
```

Comment:
```
get_limiter(&self) -> Arc<RateLimiter>
```

Reply:
can't pass compilation, the limiter has to be mutable.

Reply:
maybe i can create a pr in rust-rocksdb to fix this, i will try it.